### DISCUSSION TITLE
跟大家分享一下自己的mihomo使用模板😀

### DISCUSSION THREAD

**Discussion by refined-fish** [nikkinikki-org/OpenWrt-nikki#341] [Show and tell]
https://github.com/refined-fish/clash_rule_fish

**Comment by s1u** [comment]
proxy-providers-general
use-all-proxy-providers

这2个是什么用途？貌似没查到说明。

**Reply by refined-fish** [reply]
yaml语法的锚点，相当于变量引用，减少书写量的

**Comment by waxingren** [comment]
fakeip 不是能直接屏蔽geosite cn吗  还用hosts干啥额

**Reply by refined-fish** [reply]
写出来只是一个演示，你没发现写的是mihomo吗，方便小白需要就直接改的

**Reply by waxingren** [reply]
> 写出来只是一个演示，你没发现写的是mihomo吗，方便小白需要就直接改的

明白了   写的很好

**Comment by jervy** [comment]
谁有ai合集的规则地址呀？

**Reply by DriveFLY** [reply]
SukkaW/Surge

**Reply by waxingren** [reply]
> 谁有ai合集的规则地址呀？
https://testingcf.jsdelivr.net/gh/dler-io/Rules@main/Clash/Provider/AI%20Suite.yaml

**Reply by jervy** [reply]
这个好像不更新呀，grok也没有。不过还是谢谢你

**Comment by woni928** [comment]
你好，如果国内的某个网址走代理了，应该怎么设置？

**Reply by refined-fish** [reply]
检查你的规则是不是有问题，实在需要单独写规则就单独写一条放在有问题的规则前面（优先匹配）

**Reply by woni928** [reply]
> 检查你的规则是不是有问题，实在需要单独写规则就单独写一条放在有问题的规则前面（优先匹配）

具体怎么写？请教一下
还有如何过滤某些节点名字 比如  剩余套餐 官网订阅 长期有效 等等

**Reply by refined-fish** [reply]
直接查看Mihomo官方文档，里面写的很清楚还有示例

**Comment by shaoyongxin** [comment]
写的这么优秀，花了不少心思啊，谢谢

**Comment by ha7rock** [comment]
感谢分享，发现一个问题： yaml中的系统配置项貌似并不会覆盖掉客户端中设置的同款内容。比如在yaml中设置的fakeip-filter，tun选择的栈等等，还是使用了nikki的config文件中的设置，而这部分内容是来自客户端的配置…yaml中相关的字段的设定要如何被使用呢…

**Reply by refined-fish** [reply]
当然，请你仔细阅读我模板的开头，少部分配置代理软件是强制覆盖的，必须在软件处设置，其他部分请在代理软件处设置为“不修改”

**Reply by ha7rock** [reply]
感谢解答🙏

**Comment by shaoyongxin** [comment]
请问为什么fake-filter中我已经加了geosite:apple，appstore下载依然还是很慢

**Reply by refined-fish** [reply]
我没有苹果设备，你加入Apple规则后，DNS会发生在本地并返回真实地址。你需要排查你的直连流量吧，比如我的爱快分流架构，这种流量会直接从客户端到爱快到外网直连，我得看这个路径上那里出现了限速。你的情况也是一样的排查方法

**Reply by ssssssshi** [reply]
iosapps.itunes.apple.com
hls.itunes.apple.com.mwcname.com
DNS 重写用这个就快了

**Comment by LitCcc** [comment]
redir-host的版本还会更新吗😭

**Reply by refined-fish** [reply]
你是有啥顾虑不用Fakeip吗🫣

**Comment by thisdk** [comment]
学习到了.楼主的配置已经是极品开箱即用了.最重要并不是什么东西都乱开.只开启对路由器透明代理有利的,且注释十分详细
